openassessment/templates/openassessmentblock/oa_rubric.html

```diff
@@ -1,6 +1,6 @@
 {% spaceless %}
 {% load i18n %}
-<fieldset class="assessment__fields">
+<div class="assessment__fields">
```

Comment:
@cptvitamin said this use of fieldset is just wrong (legend must appear immediately after fieldset if it is used).